network.loki.Session.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id": "network.loki.Session",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "session",
+    "separate-locales": false,
+    "rename-desktop-file": "session-messenger-desktop.desktop",
+    "rename-icon": "session-messenger-desktop",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--own-name=org.kde.StatusNotifierItem-2-1"
+    ],
+    "modules": [
+        {
+            "name": "session-messenger-desktop",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ar x session-messenger-desktop-linux-amd64-*.deb",
+                "rm -f session-messenger-desktop-linux-amd64-*.deb",
+                "tar xf data.tar.xz",
+                "rm -f control.tar.gz data.tar.xz debian-binary",
+                "cp -r usr/* opt/* /app",
+                "rm -rf /app/share/icons/hicolor/1024x1024/",
+                "chmod -R a-s,go+rX,go-w /app/Session",
+                "sed -i 's|Exec=.*|Exec=session %U|' /app/share/applications/session-messenger-desktop.desktop",
+                "install session.sh /app/bin/session",
+                "install -Dm644 network.loki.Session.appdata.xml /app/share/appdata/network.loki.Session.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://github.com/loki-project/session-desktop/releases/download/v1.0.8/session-messenger-desktop-linux-amd64-1.0.8.deb",
+                    "sha256": "31fb4ebe74dc3b94f30136d96d164de23e81ca6cba4cb5c150e734f86e72eaad"
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "session.sh",
+                    "commands": [
+                        "export TMPDIR=\"$XDG_RUNTIME_DIR/app/$FLATPAK_ID\"",
+                        "exec zypak-wrapper /app/Session/session-messenger-desktop \"$@\""
+                    ]
+                },
+                {
+                    "type": "file",
+                    "path": "network.loki.Session.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Should be resolved as of 9437ad913ca29a7bcb534a3cd392ade48fb95079